cpp/dolfinx/fem/assemble_matrix_impl.h

```diff
@@ -428,8 +428,10 @@ void assemble_interior_facets(
     {
       for (int j = 0; j < gdim; ++j)
       {
-        coordinate_dofs(i, j) = x_g(x_dofs0[i], j);
-        coordinate_dofs(i + num_dofs_g, j) = x_g(x_dofs1[i], j);
+        // coordinate_dofs(i, j) = x_g(x_dofs0[i], j);
+        coordinate_dofs[i * gdim + j] = x_g(x_dofs0[i], j);
+        // coordinate_dofs(i + num_dofs_g, j) = x_g(x_dofs1[i], j);
```

Comment:
Leftover comments.

Reply:
I left it there deliberately to explain the operation.

Reply:
Well, seems quite obvious (just flattened array access). I would not read it as a hint, as this is a valid commented code I would treat it as some leftover.